src/kontrol/__main__.py

```diff
@@ -244,6 +244,10 @@ def exec_prove(
     if isinstance(kore_rpc_command, str):
         kore_rpc_command = kore_rpc_command.split()
 
+    for test_name, _ in tests:
+        if not test_name.startswith('test') or test_name.startswith('testFail'):
```

Comment:
```suggestion
        if not test_name.startswith('test') or not test_name.startswith('testFail'):
```

(If this is the intended condition, then the second disjunct can be dropped, see comment below.)

Reply:
Btw this forbids

```
kontrol prove --match-test '.*'
```

Is this intended?

Reply:
It wasn't, thanks for noticing!